src/com/oltpbenchmark/benchmarks/tpcc/TPCCBenchmark.java

```diff
@@ -188,6 +189,8 @@ public void createSqlProcedures() throws Exception {
           st.execute(String.format("DROP PROCEDURE IF EXISTS updatestock%d", i));
           st.execute(updateStmt);
```

Comment:
Not your code, but just to make things consistent, would you mind moving the updateStock Procedures to NewOrder.java?

Reply:
let me do this separately to avoid further merge conflicts with the other outstanding PR